keytableprinter/formatwarnings.go

```diff
@@ -4,13 +4,23 @@ import (
 	"fmt"
 
 	"github.com/fluidkeys/fluidkeys/colour"
+	"github.com/fluidkeys/fluidkeys/pgpkey"
 	"github.com/fluidkeys/fluidkeys/status"
 )
 
+func keyWarningLines(key pgpkey.PgpKey) []string {
+	keyWarnings, _ := status.GetKeyWarnings(key)
```

Comment:
If `GetKeyWarnings` *did* return an error (yes, I know I've subsequently made it never return an error!), then this function would `panic` on `range keyWarnings`...